parcels/fieldset.py

```diff
@@ -293,12 +293,69 @@ def from_nemo(cls, filenames, variables, dimensions, indices=None, mesh='spheric
         :param time_periodic: boolean whether to loop periodically over the time component of the FieldSet
                This flag overrides the allow_time_interpolation and sets it to False
         :param tracer_interp_method: Method for interpolation of tracer fields. It is recommended to use 'cgrid_tracer' (default)
-               Note that in the case of from_nemo(), the velocity fields are default to 'cgrid_velocity'
+               Note that in the case of from_nemo() and from_cgrid(), the velocity fields are default to 'cgrid_velocity'
+
+        """
+
+        fieldset = cls.from_c_grid_dataset(filenames, variables, dimensions, mesh=mesh, indices=indices, time_periodic=time_periodic,
+                                  allow_time_extrapolation=allow_time_extrapolation, tracer_interp_method=tracer_interp_method, **kwargs)
+        if hasattr(fieldset, 'W'):
+            fieldset.W.set_scaling_factor(-1.)
```

Comment:
But the warning in `AdvectionRK4_3D` is still that vertical velocity is assumed DOWNWARD; even though that is not the case with `from_nemo`. Might confuse users? Shall we change the warning message?

Reply:
This warning is still true. `AdvectionRK4_3D` assumes that positive `w` means increasing `z`.
I would not remove it. Maybe we should add "(increasing z)" after "downward", then it's more complete.

Also what we can do is to have a variable `fieldset.creation_log` or something like this.
And in `Fieldset.from_nemo` we set this variable to `from_nemo`.
And we can disable the warning if `creation_log == "from_nemo"`

Maybe we'll use this log for various purposes later on (not sure if it's a good idea though)

Reply:
> This warning is still true. `AdvectionRK4_3D` assumes that positive `w` means increasing `z`. I would not remove it. Maybe we should add "(increasing z)" after "downward", then it's more complete.

Well, if a user looks into their netcdf file, they may see something else; Since they may not realise we're already applying the `set_scaling_factor(-1)` for them, this could confuse them?

>Also what we can do is to have a variable `fieldset.creation_log` or something like this.
And in `Fieldset.from_nemo` we set this variable to `from_nemo`.
And we can disable the warning if `creation_log == "from_nemo"`

I like this idea yes. But I would do it on a per-field basis (since different `Fields` in a `FieldSet` can have a different `creation_log`, e.g. when combining models)

Reply:
Ok, I'll do it